source/help/getting-started/organizing-your-sidebar.rst

```diff
@@ -0,0 +1,176 @@
+Organizing Your Sidebar
+=======================
+
+You can organize your sidebar based on how you use Mattermost. Customizations you make to your sidebar only changes your Mattermost sidebar.
+
+Here's how your sidebar is set up by default:
+
+- All of the channels you've joined display in a **Channels** category. 
+- All channels are sorted in alphabetical order.
+- All Direct Messages display in a **Direct Messages** category below channels.
+- All Direct Messages are sorted by recent activity.
+
+What can you customize?
+-----------------------
+
+You can customize your sidebar in the following ways:
+
+- Group and order channels into categories customized just for you.
+- Mute and unmute entire categories.
+- Sort category channels manually, alphabetically, or by recent activity.
+- Group unread messages into an **Unreads** category.
+- Sort Direct Messages alphabetically or by recent activity.
+- Control how many Direct Message conversations to display.
```

Comment:
Control how many to display at once or in general?

Reply:
@justinegeffen In this case the answer is the same: both at once and in general. Do you have suggestions for clarity for this bullet point given that clarification?